packages/contentful-app-extensions/event-additional-materials/src/locations/Field.spec.tsx

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
```

Comment:
Not sure im keen on introducing a third convention on test file naming. In FE we have __test__ in be we duplicate the folder structure in a parralel directory. Now we are adding same directory under a *spec* filename?